src/workerd/io/compatibility-date.capnp

```diff
@@ -441,4 +441,13 @@ struct CompatibilityFlags @0x8f8c1b68151b6cef {
     $compatDisableFlag("fetch_legacy_url")
     $compatEnableDate("2024-06-03");
   # Ensures that WHATWG standard URL parsing is used in the fetch API implementation.
+
+  nodeJsCompatV2 @50 :Bool
```

Comment:
For visibility, should we place this next to the existing `nodeJsCompat` flag?

Reply:
I'd rather keep the ordinal values in order to make it easier to know which is the next available value.